cmd/channelz.go

```diff
@@ -26,10 +25,9 @@ func prettyTime(ts *timestamppb.Timestamp) string {
 		return ""
 	}
 	if timestampFlag {
-		return ptypes.TimestampString(ts)
+		return ts.AsTime().Format("2006-01-02T15:04:05.999999999Z07:00")
```

Comment:
Seems we should pass `time.RFC3339Nano` instead of the string, to make it clear what it is. It looks like you copied the value from `time`, although it is unclear why.